vue-js/src/components/task1/ListItem.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div id="list-item">
+    <div v-if="edit">
+      <li>{{item.text}}
+      <button v-on:click="setSavingMode">Edit</button>
+      <button  @click="$emit('delete-item')">Delete</button>
+      </li>
+    </div>
+    <div v-else>
+      <li>
+      <input id="new-item-text" v-model="item.text" />
+      <button v-on:click="saveNewText">Save</button>
+      <button v-on:click="cancelSavingMode">Cancel</button>
+      </li>
+    </div>
+    
+    
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'list-item',
+  props: ['item'],
+  data: function() {
+    return {
+      edit: true

```

Comment:
This is quite confusing: You are in edit mode when `edit: false`? :confused: 

Could you swap the meaning of this boolean, please?